net/transmission/patches/010-temp-mbedtls3-compile.patch

```diff
@@ -0,0 +1,73 @@
+From 3e8b1906f1e34b3010b8bb264435829ff7c9e74f Mon Sep 17 00:00:00 2001
+From: Seo Suchan <tjtncks@gmail.com>
+Date: Thu, 2 May 2024 03:06:42 +0900
+Subject: [PATCH] fix building with mbedtls 3.X uses renamed funtion name for
```

Comment:
This commit subject is not what is going to be applied in upstream. This patch should be equal to that.

1. https://github.com/transmission/transmission/pull/6822
2. https://github.com/transmission/transmission/pull/6823

The approach for is is to add to the end of the url .patch, so it would look like this https://github.com/transmission/transmission/pull/6823.patch